packages/vscode-extension/package.json

```diff
@@ -736,7 +736,81 @@
         ],
         "properties": {
           "command": {
-            "type": "string"
+            "oneOf": [
```

Comment:
Does this keep backward compatibility, e.g., `teamsfx: ngrok start`?

Reply:
As we discussed offline, command can be anything. The auto prompt only contains `debug-check-prerequisites`. If the command is invalid, the error message will be shown after execution.